packages/client-sdk-web/src/internal/control-client.ts

```diff
@@ -100,7 +100,7 @@ export class ControlClient<
     return await new Promise<CreateCache.Response>(resolve => {
       this.clientWrapper.createCache(
         request,
-        null,
+        this.authHeaders,
```

Comment:
just curious why you're doing the agent header in an interceptor and the auth header on the request?

Reply:
honestly not sure, maybe our interceptors just aren't working correctly? I can change it if you want.

Reply:
it's not urgent but would be nice if they were consistent.  could just file a follow-up ticket.  not a blocker.